boards/posix/native_posix/doc/board.rst

```diff
@@ -19,6 +19,8 @@ you use native host tools for compiling, debugging, and analyzing your
 Zephyr application, eliminating the need for architecture-specific
 target hardware in the early phases of development.
 
+This board provides a few peripherals. See `Peripherals`_ for more information.
```

Comment:
Some examples would be nice here:

    This board provides a few peripherals such as an Ethernet driver and UART.  See ...